include/circt/Dialect/FIRRTL/FIRRTLOpInterfaces.td

```diff
@@ -12,6 +12,59 @@
 
 include "mlir/IR/OpBase.td"
 
+def FlowKind : OpInterface<"FlowKind"> {
+  let cppNamespace = "circt::firrtl";
+  let description = [{
+    Allows operations to specify what kind of flow they have.  When an operation
+    does not implement this interface, they are assumed to have "source" flow.
+
+    If an operation has results, they should implement `getFlow`.  If they
```

Comment:
Missing reset of the description?

Reply:
Woops, I meant to delete that part.. I had a more complicated interface originally that I thought required some more detailed instructions